Packet++/header/TcpReassembly.h

```diff
@@ -240,6 +240,20 @@ class TcpReassembly
 		TcpReassemblyConnectionClosedManually
 	};
 
+	enum ReassemblyStatus 
+	{
+		Success_TcpMessageHandled,
+		Success_OutOfOrderTcpMessageBuffered,
+		Success_FIN_RSTWithNoData,
+		Ignore_IcmpPacket,
+		Ignore_PacketWithNoData,
+		Ignore_PacketOfClosedFlow,
+		Ignore_Retransimission,
+		Error_NonIpPacket,
+		Error_NonTcpPacket,
+		Error_PacketDoesNotMatchFlow,
+	};

```

Comment:
Can you please add doxygen documentation to the enum and enum values?

Reply:
I added some comments to the enum values, but I haven't used doxygen before and I didn't figure it out how to generate api-doc html on my machine. Hope it works...

Reply:
Thanks for adding the comments. Generating the html code isn't necessary, there is a CI job that does that automatically:

https://github.com/seladb/PcapPlusPlus/blob/5ef0c6da8c34b5d4155f865e94bdf83cb18fcbad/.cirrus.yml#L68